_pages/saml/authentication.md

```diff
@@ -114,8 +171,18 @@ A proofed identity request at AAL2, with phishing resistent MFA, for email, phon
 <div class="grid-row grid-gap">
     <div class="desktop:grid-col-7 mobile:grid-col-full">
         <h2 id="authentication-response">Authentication response</h2>
-        <p>After the user authenticates, Login.gov will redirect and POST a form back to your registered Assertion Consumer Service URL:</p>
-        <p>The SAMLResponse is a base64-encoded XML payload that contains encrypted data.</p>
+        <p markdown="1">After the user authenticates, Login.gov will redirect and POST a form back to your registered Assertion Consumer Service URL with a hidden form control named `SAMLResponse`.</p>
+        <p markdown="1">`SAMLResponse` contains a base64-encoded XML payload that contains data that is encrypted with the service provider's public key.</p>
+        <p markdown="1"> The decrypted `SAMLResponse` contains a `<saml:Assertion>` element, which in turn contains the following elements: </p>
+        <dl>
```

Comment:
[Praise] Description List in the wild! Thanks for using semantic HTML

Reply:
I'm still unhappy about mixing Markdown and HTML.

Reply:
I find CMSes to be consistently bizarre in their authoring choices. The ones I've seen that are very straightforward for the author tend to be shockingly expensive. (or, I guess, we could just host a wiki)